src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor

```diff
@@ -8,15 +8,28 @@
 
 <PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.ConsoleLogs.ConsoleLogsPageTitle)" Loc="@Loc" /></PageTitle>
 
-<div class="resource-logs-layout">
-    <h1 class="page-header">@Loc[nameof(Dashboard.Resources.ConsoleLogs.ConsoleLogsHeader)]</h1>
-    <FluentToolbar Orientation="Orientation.Horizontal">
+<AspirePageContentLayout AddNewlineOnDesktopToolbar="true">
```

Comment:
When I resize this page the console logs completely break.

I expect the page state (log selected, logs scroll position) to be unaffected.

![consolelogs-resize](https://github.com/dotnet/aspire/assets/303201/9d69d4c4-3ebf-4432-9753-e280a0305e0f)

Reply:
It is fairly simple to fix this for console logs, but for the pages with data grids it is not. The component is re-rendered as we are in a new layout, and the data grid does not support persisting its existing state. However, @vnbaaij noted that the next version of fluentui-blazor will have this functionality, so I will create an issue as a follow-up

Reply:
The console logs page needs changes to not lose and rewatch logs on render.

Reply:
There will be another log viewer component created on layout change. That cannot be avoided. We may be able to reduce the delay of re-showing the logs by storing previously written logs in a scoped service, but it will not be eliminated.

Reply:
The log viewer will be disposed and re-created during layout change. I don't see a way around that. We can shorten the amount of time before logs re-appear on screen but they won't stay static.